### ISSUE TITLE
Kidnapping of Boo non-functional

### ISSUE THREAD

**Issue by lambda-dom** [Pocket-Plane-Group/UnfinishedBusiness#44]
The "Kidnapping of Boo" quest is non functional for me (tested in BG2 EE). Tracked down the culprit to the last lines of `setup_kidnapping_of_boo.tpa` which has the following macro applied on copying the spells for removing/re-adding boo:

```
LPF CLONE_EFFECT
        INT_VAR
        check_global	= 0
        match_opcode	= 54
        opcode		= 206
        parameter1	= ~-1~
        parameter2	= 0
        STR_VAR
        resource	= ~%SOURCE_RES%~
      END
```

This inserts the 206 self-protection at the *beginning* of the spell, blocking its application entirely. Adding `insert = "last"` should fix the problem, e.g.:

```
LPF CLONE_EFFECT
    INT_VAR
    check_global = 0
    match_opcode = 54
    opcode = 206
    parameter1 = ~-1~
    parameter2 = 0
STR_VAR
    resource = ~%SOURCE_RES%~
    insert = "last"
END
```

**Comment by Frenzgyn** [comment]
Yep, confirm the bug, I was working on fork with the intention of of doing a PR at some point, but due to lack of time, never did. 

https://github.com/Frenzgyn/UnfinishedBusiness/commit/dde3dccf4c0a787870a8e0f6eb5cdba31e441b40
